hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -909,29 +909,17 @@ void Vgen::emit(const unpcklpd& i) {
 ///////////////////////////////////////////////////////////////////////////////
 
 void Vgen::emit(const cmpsd& i) {
```

Comment:
I think renaming the 2 argument `fcmeq` (see next comment) will help, but I also think a line or two of comments will make this more clear too.